src/libraries/System.Net.NameResolution/src/System/Net/NameResolutionPal.Unix.cs

```diff
@@ -119,16 +120,17 @@ private static unsafe void GetHostEntryForNameCallback(Interop.Sys.HostEntry* en
             // Can be casted directly to GetHostEntryForNameContext* because the HostEntry is its first field
             GetHostEntryForNameContext* context = (GetHostEntryForNameContext*)entry;
 
-            ProcessResult(GetSocketErrorForNativeError(error), context);
+            ProcessResult(error, context);
         }
 
-        private static unsafe void ProcessResult(SocketError errorCode, GetHostEntryForNameContext* context)
+        private static unsafe void ProcessResult(int error, GetHostEntryForNameContext* context)
```

Comment:
Why this change?  We can't continue to use SocketError in the presence of EAI_CANCELED?  Can't it map to one of the existing SocketError values?

Reply:
I didn't find a matching value in https://github.com/dotnet/runtime/blob/f6e73ad8c8a3252a5528724fa241fca410295008/src/libraries/System.Net.Primitives/src/System/Net/Sockets/SocketError.cs#L9

Reply:
SocketError.OperationAborted

Reply:
Ey, so easy...thanks.